asciidoc/runtime/src/main/java/io/quarkiverse/qute/web/asciidoc/runtime/AsciidocConverter.java

```diff
@@ -13,11 +13,45 @@ public class AsciidocConverter {
             .setAttributes(Map.of("noheader", "true"));
 
     public String apply(String asciidoc) {
-        Body body = parser.parseBody(asciidoc, new Parser.ParserContext(null));
+        final String content = trimIndent(asciidoc);
+        Body body = parser.parseBody(content, new Parser.ParserContext(null));
         // Renderer is not thread-safe and must not be shared
         AsciidoctorLikeHtmlRenderer renderer = new AsciidoctorLikeHtmlRenderer(config);
         renderer.visitBody(body);
         return renderer.result();
     }
 
+    public static String trimIndent(String content) {
```

Comment:
Is this a workaround for https://github.com/yupiik/tools-maven-plugin/issues/21? If so, shouldn't we add a comment here?

Reply:
I added a comment, whatever the fix is, we should keep it as explained in the comment.